08_calculator/solution/calculator-solution.js

```diff
@@ -10,11 +10,13 @@ const sum = function (array) {
   return array.reduce((total, current) => total + current, 0);
 };
 
-const multiply = function (array) {
-  return array.length
-    ? array.reduce((accumulator, nextItem) => accumulator * nextItem)
-    : 0;
-};
+const multiply = function(...args){
+  let sum = 1
```

Comment:
very nitpicky: can we call this variable `product`

"product" is the result of multiplying numbers.. "sum" is for adding.  this doesn't matter toooo much, but since this is example code we're going to have thousands of students looking at... we want it to be as clean and correct as possible.

Reply:
Good point; no problem, will do. Thanks!